src/Aggregrid.js

```diff
@@ -378,7 +378,7 @@ Ext.define('Jarvus.aggregrid.Aggregrid', {
             );
         }
 
-        if (target = ev.getTarget('.jarvus-aggregrid-cell', containerEl, true)) { // eslint-disable-line no-cond-assign
+        if ((target = ev.getTarget('.jarvus-aggregrid-cell', containerEl, true)) && target.up('.jarvus-aggregrid-row')) { // eslint-disable-line no-cond-assign
```

Comment:
@themightychris subcells do not have a `.jarvus-aggregrid-row` as a parent, they have a `.jarvus-aggregrid-subrow` as a parent